homeassistant/components/unifi/hub/entity_loader.py

```diff
@@ -43,10 +46,24 @@ def __init__(self, hub: UnifiHub) -> None:
             hub.api.port_forwarding.update,
             hub.api.sites.update,
             hub.api.system_information.update,
+            hub.api.traffic_rules.update,
             hub.api.wlans.update,
         )
+        self.polling_api_updaters = (hub.api.traffic_rules.update,)
         self.wireless_clients = hub.hass.data[UNIFI_WIRELESS_CLIENTS]
 
+        self._dataUpdateCoordinator = DataUpdateCoordinator(
+            hub.hass,
+            LOGGER,
+            name="Unifi entity poller",
+            update_method=self._update_pollable_api_data,
+            update_interval=POLL_INTERVAL,
+        )
+
+        self._update_listener = self._dataUpdateCoordinator.async_add_listener(
+            update_callback=lambda: None
+        )
```

Comment:
Is this relevant?

Reply:
Without `update_callback` the update coordinator won't start the schedule for refreshing data.